code/game/objects/items/weapons/tools.dm

```diff
@@ -721,6 +721,57 @@
 	playsound(get_turf(src), 'sound/medbot/Flatline_custom.ogg', 35)
 	return (SUICIDE_ACT_BRUTELOSS)
 
+/obj/item/tool/crowbar/halligan
+	name = "Halligan bar"
+	desc = "Combination pick, crowbar, and adze used for forcible entry."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "halligan"
+	item_state = "halligan"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/items_lefthand.dmi', "right_hand" = 'icons/mob/in-hand/right/items_righthand.dmi')
+	hitsound = "sound/weapons/toolhit.ogg"
+	item_state = "halligan"
+	w_class = W_CLASS_MEDIUM
+	attack_verb = list("pries", "slashes", "stabs", "bludgeons", "whacks")
+
+/obj/item/tool/crowbar/halligan/attackby(obj/item/I, mob/user)
+	if(istype(I,/obj/item/weapon/fireaxe))
+		var/obj/item/weapon/fireaxe/F = I
+		to_chat(user, "<span class='notice'>You attach \the [F] and [src] to carry them easier.</span>")
+		var/obj/item/tool/irons/SI = new (user.loc)
+		SI.fireaxe = F
+		SI.halligan = src
+		user.drop_item(F)
+		F.forceMove(SI)
+		user.drop_item(src)
+		forceMove(SI)
+		user.put_in_hands(SI)
+		return 1
+	return 0
+
+/obj/item/tool/irons
+	name = "set of irons"
+	desc = "Fireaxe and Halligan bar used for forcible entry."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "irons"
+	hitsound = "sound/weapons/toolhit.ogg"
+	item_state = "irons"
+	w_class = W_CLASS_MEDIUM
```

Comment:
Why does strapping a halligan bar to the fireaxe make the fireaxe go down 1 size class?

Reply:
Meant for this to be easier to carry in combined form, but wearing it on your back or suit slot already does this so I'll bring it back to large

Reply:
Resolved in latest commit